scripts/install-cni.sh

```diff
@@ -121,15 +121,24 @@ if [ "$ENABLE_MASQUERADE" == "true" ]; then
   fi
 fi
 
-if [ "$ENABLE_PRIVATE_IPV6_ACCESS" == "true" ]; then
+if [ "$ENABLE_PRIVATE_IPV6_ACCESS" == "true" ] || [ "$ENABLE_IPV6" == "true" ]; then
   node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/?recursive=true" -H "Metadata-Flavor: Google" | jq -r '.ipv6s[0]' ) ||:
 
   if [ -n "${node_ipv6_addr:-}" ] && [ "${node_ipv6_addr}" != "null" ]; then
```

Comment:
Just double checking - does the same `node_ipv6_addr` checks make sense for dualstack? (Asking as I don't see `node_ipv6_addr` being used anymore.)

Reply:
When enable_ipv6 is true, we need to have a valid node_ipv6_addr. So the node_ipv6_addr checks are still valid.
Just the way we get ipv6Subnet will be different.